setup.py

```diff
@@ -1,42 +1,6 @@
 from __future__ import absolute_import, division, print_function
 
-from setuptools import find_packages, setup
-
-# Long description will go up on the pypi page
-with open("README.md") as file:
-    LONG_DESCRIPTION = file.read()
-
-# Dependencies.
-with open("requirements.txt") as f:
```

Comment:
So, using setup.py allowed the reading of requirements from the requirements files, which minimized redundant dependency listings. But in the switch to setup.cfg, that capability is not longer there?

Reply:
No, setup.cfg is static. Though we can still keep it dynamic with setup.py.

Reply:
Okay. I've updated this now so that we have the dynamic generation. Let me know what you think.